integration/spark2/src/test/scala/org/apache/spark/carbondata/commands/SetCommandTestCase.scala

```diff
@@ -128,6 +128,34 @@ class SetCommandTestCase extends Spark2QueryTest with BeforeAndAfterAll{
         sql(s"set ${CarbonLoadOptionConstants.CARBON_OPTIONS_SINGLE_PASS}"))
     }
   }
+
+  test(s"test set ${ CarbonLoadOptionConstants.CARBON_TABLE_LOAD_SORT_SCOPE } for valid options") {
+    checkAnswer(
+      sql(s"set ${ CarbonLoadOptionConstants.CARBON_TABLE_LOAD_SORT_SCOPE }=no_sort"),
```

Comment:
As the check property is being validated to have dbname and tablename in CarbonHiveCommands.scala therefore this test case should have failed. 

There is a problem in https://github.com/apache/carbondata/pull/3014/files#diff-eeaa616740e33846839055324b58aa4bR87 as all the properties with @CarbonProperty annotations are being added blindly.

Please fix this.

Reply:
Done.